### ISSUE TITLE
document-title rule should not fail framed pages

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1424]
Got a report of axe-core calling out the content of the following iframe:

```html
<iframe tabindex="-1" role="presentation" src="..."></iframe>
```

Axe-core shouldn't fail iframes for SC3.1.1. It isn't applicable to framed pages, only to the top window. While it might be true that such pages can be displayed without the frame, they aren't on the page that is tested, so I don't think axe-core should ever report them.

**Comment by WilcoFiers** [comment]
There's an easy fix for this. We can add the `window-is-top.js` matcher to the document-title rule.

**Comment by jkodu** [comment]
@WilcoFiers 

I believe you intended to say rule `html-has-lang (sc311)` and not `document-title (sc242)`.
